lib/GameBLoC.dart

```diff
@@ -67,6 +67,7 @@ class GameBLoC {
 
   login() async {
     _signedIn = (await GamesServices.signIn()) == "success";
+    print(_signedIn);
```

Comment:
I'd clean up some prints, and leave what's remaining as verbose and as debugPrints